src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 20px;
+  background-color: black;
+}
+
+.outer {
+  margin-bottom: 40px;
+  background-color: #308dc0;
+  border: 1px solid #e95d49;
+  width: 348px;
+}
+
+.middle {
+  box-sizing: border-box;
```

Comment:
also it's better to add box-sizing: border-box rule for every element on the page (you can use * for this)